### PR TITLE
fix closing of context

### PR DESCRIPTION
**Pull request by Simon-Laux**
this should solve the event emitter problem leading to crashes

### CODE REVIEW

**Review by link2xt — APPROVED** [review]

**Comment by link2xt** [comment]
Why is the context closed immediately after app startup? Does desktop app create new context, then looks up existing accounts, closes temporary context and opens context for existing account?

**Comment by Simon-Laux** [comment]
desktop opens contexts and closes to get the account avatar picture name and email address:
https://github.com/deltachat/deltachat-desktop/blob/master/src/main/logins.ts

File: src/module.c [hunk 1]
```diff
@@ -299,6 +299,7 @@ NAPI_METHOD(dcn_context_unref) {
   NAPI_DCN_CONTEXT();
 
   TRACE("Unrefing dc_context");
+  dcn_context->gc = 1;
```
Comment (Jikstra) [inline src/module.c:302]: Can you describe why this fixed the problem? ^^

Comment (Simon-Laux) [reply src/module.c:302]: because the event loop checks for it and you forgot to put it back while you were refactoring it a long time ago.

Comment (Simon-Laux) [reply src/module.c:302]: line 176 if you wonder

**Pull request merged**